### PR TITLE
Changes to work with latest version of ember-inspector

### LINKED ISSUE #8: Cannot find module 'express'
Ember-cli 0.2.0 removes express from package.json, and this addon assumes it'll be there.. so somehow this addon needs to get express into the picture

(Well, technically it's the `ember init` from ember-cli 0.2.0 that removes this)

### PR DESCRIPTION
**Pull request by joostdevries**
Check https://github.com/emberjs/ember-inspector/pull/280.
Will put in the correct version number and release once a new version of the inspector is released.

also fixes #8

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
   "license": "MIT",
   "dependencies": {
     "socket.io":"1.1.0",
-    "ember-inspector": "^1.6.3"
+    "ember-inspector": "emberjs/ember-inspector#master"
```
Comment (teddyzeenny) [inline package.json]: This should be `1.7.0` (not released yet)

**Comment by teddyzeenny** [comment]
:+1: This will be merged when we release Ember Inspector 1.7 right?

**Comment by joostdevries** [comment]
@teddyzeenny yup. As soon as you release, I'll put the correct version in package.json and release a new version too.

**Comment by teddyzeenny** [comment]
New version released.

**Comment by joostdevries** [comment]
@teddyzeenny Yay! I'm going to release this weekend. 
- [ ] Want to add a few basic tests first.

File: package.json (hunk 1 above)
Comment (teddyzeenny) [inline package.json]: `1.8.0` released :)

Comment (joostdevries) [inline package.json]: I SUCK. Sorry :-( I've been struggling to find time to work on ember-related stuff the past months :-(

**Comment by joostdevries** [comment]
@teddyzeenny would love your thoughts on this..

**Comment by teddyzeenny** [comment]
Yay!! Lgtm :smile:

**Pull request merged**